packages/js/src/general/components/alert-items/ping-other-admins-alert-item.js

```diff
@@ -0,0 +1,208 @@
+/* eslint-disable complexity */
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import { ArrowNarrowRightIcon } from "@heroicons/react/outline";
+import { select, useDispatch } from "@wordpress/data";
+import { useState, useCallback, useRef } from "@wordpress/element";
+import { __, sprintf } from "@wordpress/i18n";
+import { isEmail } from "@wordpress/url";
+import { Button, TextField } from "@yoast/ui-library";
+import { STORE_NAME } from "../../constants";
+import { useSelectGeneralPage } from "../../hooks";
+import { safeCreateInterpolateElement } from "../../../helpers/i18n";
+
+/**
+ * A function to send a request to the mailing list API.
+ *
+ * @param {string} email The email to signup to the newsletter.
+ * @param {boolean} isPremium Whether Premium is active.
+ * @param {boolean} isWooSeoActive Whether Woo SEO is active.
+ *
+ * @returns {Object} The request's response.
+ */
+async function mailingListSubscribe( email, isPremium, isWooSeoActive ) {
+	let source;
+	if ( isWooSeoActive ) {
+		source = "woo-seo";
+	} else if ( isPremium ) {
+		source = "premium";
+	} else {
+		source = "free";
+	}
+
+	// eslint-disable-next-line no-warning-comments
+	// @TODO Change to production URL before merging
+	const mailingListResponse = await fetch( "https://staging-my.yoast.com/api/Mailing-list/subscribe", {
+		method: "POST",
+		headers: {
+			"Content-Type": "application/json",
+		},
+		body: JSON.stringify( {
+			customerDetails: {
+				firstName: "",
+				email: email,
+			},
+			list: "Yoast newsletter",
+			source: [ source, "recapture" ],
+		} ),
+	} );
+
+	return mailingListResponse.json();
+}
+
+/**
+ * A function to request permanently resolving the alert.
+ *
+ * @param {string} id The alert ID.
+ * @param {string} resolveNonce The nonce to resolve the alert.
+ *
+ * @returns {Object} The request's response.
+ */
+async function resolveAlert( id, resolveNonce ) {
+	const formData = new FormData();
+	formData.append( "action", "wpseo_resolve_alert" );
+	formData.append( "alertId", id );
+	formData.append( "_ajax_nonce", resolveNonce );
+
+	const ajaxUrl = select( STORE_NAME ).selectPreference( "ajaxUrl" );
+
+	const resolveResponse = await fetch( ajaxUrl, {
+		method: "POST",
+		body: formData,
+	} );
+
+	return resolveResponse.json();
+}
+
+/**
+ * Ping other admins alert item component.
+ *
+ * @param {Object} props Component props.
+ * @param {string} props.id Alert ID.
+ * @param {boolean} props.dismissed Whether the alert is dismissed.
+ * @param {string} props.message Alert message.
+ * @param {string} props.resolveNonce Nonce to resolve the alert.
+ * @returns {JSX.Element} The PingOtherAdminsAlertItem component.
```

Comment:
Not a big deal, but we usually don't add `prop.` in the param description.

Reply:
Done [here](https://github.com/Yoast/wordpress-seo/pull/22672/commits/b6a84e0c7379d7a27e3f3fb2dc2ae32b13298c4f).